core/src/main/java/io/confluent/kafka/schemaregistry/storage/KafkaSchemaRegistry.java

```diff
@@ -988,7 +825,7 @@ private boolean isSubjectVersionDeleted(String subject, int version)
     }
   }
 
-  int getMaxIdInKafkaStore() {
+  public int getMaxIdInKafkaStore() {
```

Comment:
If SchemaRegistry interface is being made available to the Master Elector, would it make sense to make an interface like getMaxIdInStore() ?